app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -9584,6 +9584,23 @@ class BaseInfo
             :arovane_beat_d,
             :arovane_beat_e
           ]},
+
+        :tbd => {
+          :desc => "Black Dog samples",
```

Comment:
```suggestion
          :desc => "The Black Dog samples",
```
For consistency with the prefix (tbd_)